Formula/p/percona-toolkit.rb

```diff
@@ -45,8 +43,8 @@ class PerconaToolkit < Formula
   end
 
   resource "DBD::mysql" do
-    url "https://cpan.metacpan.org/authors/id/D/DV/DVEEDEN/DBD-mysql-5.002.tar.gz"
-    sha256 "8dbf87c2b5b8eaf79cd16507cc07597caaf4af49bc521ec51c0ea275e8332e25"
+    url "https://cpan.metacpan.org/authors/id/D/DV/DVEEDEN/DBD-mysql-5.004.tar.gz"
```

Comment:
not quite sure why it is 5.004 rather than 5.005.

Reply:
5.004 is the latest version? https://github.com/perl5-dbi/DBD-mysql/releases

Reply:
somehow this commit did not get tagged to 5.005, https://github.com/perl5-dbi/DBD-mysql/commit/21e9ef2cd0d8439944003d32bcbd592ccb8326d7

Reply:
Not sure if it affects other functionality, but the functions causing previous build failures where patched in 5.004 https://github.com/perl5-dbi/DBD-mysql/pull/415